MonumentAddons.cs

```diff
@@ -4441,6 +4442,68 @@ protected virtual void PostEntitySpawn()
                     spooker.SetFlag(BaseEntity.Flags.Busy, true);
                 }
 
+                var telephone = Entity as Telephone;
+                if (telephone != null && telephone.prefabID == 1009655496)
+                {
+                    string phoneName = null;
+                    var gridCoodr = PhoneController.PositionToGridCoord(telephone.Controller.baseEntity.transform.position);
+
+                    var monumentInfo = Monument.Object as MonumentInfo;
+                    if (monumentInfo != null && !string.IsNullOrEmpty(monumentInfo.displayPhrase.translated))
+                    {
+                        phoneName = monumentInfo.displayPhrase.translated;
+
+                        var monuments = PluginInstance.FindMonumentsByShortName(Monument.ShortName);
+                        if (monuments != null && monuments.Count > 1)
+                        {
+                            phoneName += " " + gridCoodr;
+                        }
+                    }
+
+                    var dungeonGridCell = Monument.Object as DungeonGridCell;
+                    if (dungeonGridCell != null)
+                    {
+                        phoneName = "FTL " + Regex.Replace(Monument.AliasOrShortName, "([a-z])([A-Z])", "$1 $2");
```

Comment:
I believe 'Freight Transit Line' is little easter egg joke from Facepunch, as FTL generally stands for 'Faster than light'. I tought it would be nice to add this to phone names as prefix, instead of words like 'Metro' or 'Underground' and help spread the fun :)

Reply:
Regex is now compiled, but Im not sure where is best place for its initialization, so I added it as plugin property for now.

I added attached monument name to train stations. I can't imagine how pseudo-monuments would work, but since we know what monument is station attached to, we can let user set `SpawnOnThisStationOnly` option per entity, save imonument shortname to profile and before spawn, check if attached monument shortname matches saved one. This would allow having both 'global' and 'station-specific' entities at same time.